pom.xml

```diff
@@ -334,7 +334,12 @@
             <dependency>
                 <groupId>com.google.guava</groupId>
                 <artifactId>guava</artifactId>
-                <version>20.0</version>
+                <version>33.4.8-jre</version>
```

Comment:
Nostin version uusimpaan, kun pitäisi olla taaksepäin yhteensopiva: https://github.com/google/guava/wiki/Compatibility#backward-compatibility